swap/src/alice/swap.rs

```diff
@@ -102,6 +103,7 @@ pub async fn swap(
             s_a,
             v_a,
         } => {
+            info!("Alice State: started");
```

Comment:
These log statements can probably go once the punish test PR is merged

Reply:
yeah, we can potentially drop those. It would be nice to print a bit more information than in the publish tests though. I see them as output that should help a user to understand what already happened.